US-CBP/cbp-theme
# Gap update

#### What's this PR do?
Removes row gap from CSS Grid

#### Where should the reviewer start?
Download and run branch locally.

#### How should this be manually tested?
It is easier to view in Firefox, once it's run locally change the main div from "cbp-ds-grid" to "cbp-grid" and the gaps in row should be removed. You should only see a gap in the columns.

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
- Is there a blog post?
N/A

- Does the knowledge base need an update?
N/A
